### DISCUSSION TITLE
Sub sub menu ?

### DISCUSSION THREAD

**Discussion by RenaudILTIS** [danpros/htmly#805] [General]
In HTMLy, you can create a sub-menu, but is it possible to create a sub-sub-menu? If so, how? If not, is it possible to implement it? (There's a breadcrumb trail, but with only two levels, it looks a bit weird).

**Comment by Joduai** [comment]
afaik default functionality doesn't support 3rd lvl menu nesting.  

Although I think it could be possible using menu editor, adding certain classes to elements, and using `menu($custom = true) ` funct along with creating ul>li lists depending on item classes.

Long time ago I was wondering if it could be possible to nest elements with a flat ul list, but the only idea I could come up was incorporating jscript/jq.  You can check this fiddle to get the overall idea of dropdown using 1 flat ul
https://jsfiddle.net/ry6mn8er/1/

**Comment by RenaudILTIS** [comment]
Thanks @Joduai, but too smart for me 😄

**Comment by Joduai** [comment]
tbh I wasn't even aware  that htmly lacks such function until you asked. I assumed 3rd and deeper lvl nesting could be at least achieved with custom menu.  Might be a good idea for a feature, allowing this cms to become a modern small/lightweight websites manager which doesn't require blog/news (generally dynamic article) management.  
For such basic functioning it requires calling only couple functions in a template.

Although it would require providing a simple non-blog template, as some sort of base for such static websites. All currently available themes are made for blogs.


edit: looks like there's also a bug in sitemap creation function for static pages, as it contains only 1lvl static pages, without subpages
